suricata.yaml.in

```diff
@@ -1100,6 +1100,9 @@ asn1-max-frames: 256
 #  group: suri
 
 security:
+  # if true, prevents process creation from Suricata by calling
+  # setrlimit(RLIMIT_NPROC, 0)
+  limit-noproc: yes
```

Comment:
nit: we state "if true", but then the config value is set to "yes". Better to use same word to avoid confusion...

Reply:
yaml... ok